newIDE/app/src/EventsSheet/EventsTree/ClassNames.js

```diff
@@ -24,6 +24,8 @@ export const eventsTreeWithSearchResults = 'with-search-results';
 
 export const dropIndicator = 'drop-indicator';
 export const cantDropIndicator = 'cant-drop-indicator';
+export const eventDropIndicator = 'event-drop-indicator';
```

Comment:
Should we actually reuse the existing dropIndicator?

Reply:
Finally I used the same color that is used for closable tabs and object lists. (I added dropIndicator in the theme)
Regarding the size, it may not be lisible enough to keep the same everywhere (the one in object list is quite thin).

It may be good to create a DropIndicator in the UI folder in the future (currently, we have 3-4 drop indicators, that are all implemented in a different way 😅 )